client/src/components/FavCheckbox.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from '@emotion/styled';
+// import CheckmarkPath from '../icons/CheckmarkIcon';
+
+const CheckboxContainer = styled.label`
+  display: flex;
+  width: 20px;
+  height: 20px;
+  background-color: ${props => props.theme.colors.quinary};
+  border-radius: 5px;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  margin: 1px;
+  padding: 1.5px;
+  cursor: pointer;
+`;
+
+const StyledCheckbox = styled.input`
+  opacity: 0.55;
+  display: flex;
+  width: 14px;
+  height: 14px;
+  border: none;
+  justify-content: center;
+  align-items: center;
+`;
+
+function FavCheckbox(props) {
+  const [checked, setChecked] = React.useState(undefined);
```

Comment:
I'd go with `false` as initial state, that way you know it's got to be of type `boolean`

Reply:
I changed that, thank you.